README.md

```diff
@@ -4,6 +4,26 @@
 
 This package contains the ROS2 message definitions of the [PX4 Pro ecosystem](https://px4.io/). Building this package generates all the required interfaces to interface ROS2 nodes with the PX4 autopilot internals, which use the [uORB messaging API](https://dev.px4.io/en/middleware/uorb.html). Currently the messages of this package represent a dependency to [`px4_ros_com` package](https://github.com/PX4/px4_ros_com).
 
+
+### Copy the deb packages to your local directory
+```bash
+docker create --name px4-msgs-artifacts ghcr.io/tiiuae/tiiuae/tii-px4-msgs:main
+mkdir -p /tmp/px4_msgs_debs/bin
+docker cp px4-msgs-artifacts:/artifacts/. /tmp/px4_msgs_debs/bin/
+docker rm px4-msgs-artifacts
+ls -la /tmp/px4_msgs_debs/bin
+```
+
+### Copy and install deb packages in Dockerfile
```

Comment:
Maybe document here what is the usage of the packages built in this repo.

Reply:
Yes, I was going to suggest the same place :+1:

I will wait for Jari's comments as well, then fix/add them together.